app/src/main/java/com/vitorpamplona/amethyst/service/relays/Client.kt

```diff
@@ -27,13 +27,21 @@ object Client: RelayPool.Listener {
     private var subscriptions = mapOf<String, List<TypedFilter>>()
 
     @Synchronized
-    fun connect(relays: Array<Relay>) {
+    fun connect(relays: Array<Relay>, searchRelays: List<Relay>) {
         RelayPool.register(this)
         RelayPool.unloadRelays()
         RelayPool.loadRelays(relays.toList())
+        RelayPool.loadSearchRelays(searchRelays)
         this.relays = relays
```

Comment:
What if instead of an extra parameter, we had a new RelaySet Type for search? It would be similar to the Home, DM, Public Chat, Global split we have today, with an extra one for Search.

Reply:
I guess that's definitely the way to go in the future, but not sure it makes sense atm. There is only one such relay. 'Search' icon would occupy space in all other relays for nothing. Plus I didn't want users to be forced to have this relay added to their relay set, the way it's done now is that it's relied on only for search. 

Another way could be to have a 'search relay' setting that's pre-filled with this relay, and users could type another one, etc?